### PR TITLE
updating link to code of conduct and security docs to org

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

https://github.com/opencontainers/org/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ You can find official releases of `runc` on the [release](https://github.com/ope
 
 ### Security
 
-If you wish to report a security issue, please disclose the issue responsibly
-to security@opencontainers.org.
+If you wish to report a security issue, please see the protocol under [security](security).
```
Comment (cyphar) [inline README.md]: Should be `[security]`.

Comment (vsoch) [reply README.md]: okay I think I got it right this time.

**Comment by cyphar** [comment]
Please squash your commits.

File: README.md (hunk 1 above)
Comment (cyphar) [reply README.md]: That works, as would `[security][security]`. :P

**Comment by vsoch** [comment]
Can't we squash on merge?

**Comment by cyphar** [comment]
We can, but I don't like doing it because it screws up the authors GPG signatures. But if you'd prefer that, I can do it (I do merge commits locally because I want to sign those too).

**Comment by vsoch** [comment]
I definitely can - I’ve never squashed locally before. How do I do it?

**Comment by cyphar** [comment]
The most generic way to do it is with an interactive git rebase (`git rebase -i`). Effectively you would do:

```
% git rebase -i origin/master
```

which will open your text editor, with a bunch of lines like this:

```
pick c6297bdbfacf updating link to code of conduct and security docs to org
pick defb213bc44c fixing security link
pick 89c3a9d3f036 fixing code of conduct link
```

Each line is one commit, in order -- and the idea is that the interactive rebase will apply each line in-order (`pick` just means to use that commit -- it comes from `git cherry-pick`).

This allows you to mess with the commits individually (you could re-order them or edit them), but since we want to squash them we can use `squash` -- but we also don't care about the commit messages for fix-up patches so we can just use `fixup` to avoid writing a new commit message. So you'd change the above lines to (you can use `f` instead of `fixup` if you want):

```
pick c6297bdbfacf updating link to code of conduct and security docs to org
fixup defb213bc44c fixing security link
fixup 89c3a9d3f036 fixing code of conduct link
```

Which will translate to:

1. Use the commit c6297bdbfacf.
2. Squash defb213bc44c into the previous commit (c6297bdbfacf) to create a new commit `$foo`.
3. Squash 89c3a9d3f036 into the previous commit (`$foo`) to create a new commit `$bar`.

And finally your current branch will now be `$bar`. If it's still a bit confusing, you could look up some tutorials online about interactive rebasing.

**Comment by vsoch** [comment]
Cool I've never done that - this looks fun! Be back after futzing around a bit :)

**Comment by vsoch** [comment]
okay I'm not sure I catch the $foo $bar mapping to fix/CoC-link and origin/master, but I did something and I'm going to try pushing it. Worst case scenario I'll just start over.

**Comment by vsoch** [comment]
Nope, definitely failed. Can you explain the $foo $bar mapping?

**Comment by vsoch** [comment]
Let's use this one instead https://github.com/opencontainers/runc/pull/1997

**Comment by cyphar** [comment]
You don't need to worry about `$foo` or `$bar` -- you just need to change the lines like I said. The merge commit you have here is probably because you did a `git pull`. Fixing this is much less fun (I personally have `git pull --rebase` set up to be done by default, because it leads to less issues like that).

**Pull request closed without merging**